hacks/install_etcdctl

```diff
@@ -21,27 +21,36 @@ function show_help () {
 }
 
 function install () {
-  local version=$1
+  local version="${1:-v3.4.30}" # default version "v3.4.3
```

Comment:
I was thinking about setting the default version to the latest from `etcd` github release, but I saw that they are tagging latest released as `latest` and not the highest version (version `3.4.xx` may get tagged as higher than `3.5.xx`)

Reply:
we could let [renovate](https://github.com/gardener/ops-toolbelt/pull/107) manage the version. I just tested it and it took `v3.5.12` instead of `v3.4.31`, which is currently tagged as latest
![Screenshot 2024-03-25 at 09 52 13](https://github.com/gardener/ops-toolbelt/assets/5526658/0ead7f58-765c-49de-bbbb-d069a9b30ee5)

Reply:
Should I adopt the renovate on boarding PR as well?

Reply:
no, either this PR gets merged first THEN the onboarding PR can be adapted, or the onboarding PR gets merged first then we can adapt this PR